rust-sgx-sdk/version

```diff
@@ -1 +1 @@
-565960cd7b4b36d1188459d75652619971c43f7e
+e8a9fc22939befa27ff67f5509b2c2dfe8499945
```

Comment:
I'd recommend to these kind of upgrades in a separate PR. We should also update the github dependencies if we update the version